snakemake_executor_plugin_aws_batch/__init__.py

```
@@ -204,112 +132,37 @@ def run_job(self, job: JobExecutorInterface):
+                job_command=remote_command,
+                batch_client=self.batch_client,
+            )
+            job_info = job_definition.submit()
             self.logger.debug(
                 "AWS Batch job submitted with queue {}, jobId {} and tags {}".format(
-                    self.settings.job_queue, submitted["jobId"], self.settings.tags
+                    self.settings.job_queue, job_info["jobId"], self.settings.tags
                 )
             )
         except Exception as e:
             raise WorkflowError(e)
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)